test/integration/infrastructure/infrastructure_test.go

```diff
@@ -535,6 +624,7 @@ func prepareVPC(ctx context.Context, clientFactory alicloudclient.ClientFactory,
 	vpcClient, err := clientFactory.NewVPCClient(region, *accessKeyID, *accessKeySecret)
 	Expect(err).NotTo(HaveOccurred())
 	createVpcReq := vpc.CreateCreateVpcRequest()
+	createVpcReq.VpcName = "provider-alicloud-infra-test"
```

Comment:
what will happen if there are several testing running in parallel? will this name cause any conflict?